pkgs/os-specific/darwin/apple-source-releases/system_cmds/meson.build

```diff
@@ -0,0 +1,406 @@
+# Project settings
+project('system_cmds', 'c', version : '@version@')
+
+if host_machine.system() != 'linux'
+  add_project_arguments('-D__FreeBSD__', language : 'c')
+endif
+
+sdk_version = get_option('sdk_version')
+
+# Dependencies
+cc = meson.get_compiler('c')
+fake_gperf = find_program('awk', required : true)
+
+## Frameworks
+cfopen_directory = dependency('appleframeworks', modules : 'CFOpenDirectory')
+core_foundation = dependency('appleframeworks', modules : 'CoreFoundation')
+core_symbolication = dependency('appleframeworks', modules : 'CoreSymbolication')
+directory_service = dependency('appleframeworks', modules : 'DirectoryService')
+iokit = dependency('appleframeworks', modules : 'IOKit')
+kernel = declare_dependency(include_directories : '@kernel@/Library/Frameworks/Kernel.framework/Headers/')
+open_directory = dependency('appleframeworks', modules : 'OpenDirectory')
+
+## Libraries
+ncurses = cc.find_library('ncurses')
+openbsm = cc.find_library('bsm')
+pam = cc.find_library('pam')
```

Comment:
i thought that find library looks for things in normal system places, like /, but dependency will use pkg-config and cmake (if available).

Reply:
ncurses has pkg-config support, but neither OpenBSM nor PAM do. I’ve updated the PR to use `dependency` to find ncurses.